.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    'plugin:vue/vue3-essential',
```

Comment:
使うのであればvue2のものを使う。
```suggestion
    'plugin:vue/recommended',
```

参考: https://eslint.vuejs.org/user-guide/